srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -30,10 +30,11 @@ global
     maxconn     {{ (150 / salt['pillar.get']('haproxy:nbproc', 2)) | round(0, 'ceil') | int }}
     user        haproxy
     group       haproxy
+    nbproc	1
     {%- if "physical" in grains['virtual'] %}
-    nbproc      {{ salt['pillar.get']('haproxy:nbproc', 2) }}
+    nbthread     {{ salt['pillar.get']('haproxy:nbthread', 16) }}
     {%- else %}
-    nbproc      2
+    nbthread	1
```

Comment:
Get the nbthread value along with if/else for HW/VM from pillar.

Reply:
Remove if/else blocks in haproxy config.
```
{%- if "physical" in grains['virtual'] %}
    nbthread     {{ salt['pillar.get']('haproxy:nbthread', 16) }}
{%- else %}
    nbthread	1
{% endif %}
```
becomes
```
nbthread    {{ pillar['haproxy']['nbthread'] }}
```